dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -1,8 +1,5 @@
-from __future__ import print_function
-
 import os
 import unittest
-from cgi import test
```

Comment:
@JGSweets was this an important part specifically for graph somehow, though?

Reply:
@taylorfturner My assumption is this was left over from a previous test looking through visual output reports at one point for verification.

I think it is fine to remove since we aren't using it.